src/php_pqres.h

```diff
@@ -28,6 +28,7 @@ typedef enum php_pqres_fetch {
 #define PHP_PQRES_CONV_SCALAR	0x000f
 #define PHP_PQRES_CONV_ARRAY	0x0010
 #define PHP_PQRES_CONV_DATETIME	0x0020
+#define PHP_PQRES_CONV_BYTEA	0x0030
```

Comment:
This cannot be `0x0030`, because that would equal `(PHP_PQRES_CONV_ARRAY|PHP_PQRES_CONV_DATETIME)`.
Also, I'd consider it a scalar, so `0x0008` should do it.